source/extensions/filters/http/cache/cache_filter.cc

```diff
@@ -126,6 +123,17 @@ Http::FilterDataStatus CacheFilter::encodeData(Buffer::Instance& data, bool end_
   return Http::FilterDataStatus::Continue;
 }
 
+void CacheFilter::onAboveWriteBufferHighWatermark() { high_watermark_calls_++; }
```

Comment:
Optional: I know it only matters for class types, but it's nice to stick to preincrement/predecrement so there's one less thing to think about (here and in onBelowWriteBufferLowWatermark)